src/plugins/intel_gpu/src/runtime/layout.cpp

```diff
@@ -623,4 +617,32 @@ ov::PartialShape layout::transform(cldnn::format new_fmt) const {
     return ov::PartialShape(new_shape);
 }
 
+// Check a reorder is 1d along feature axis. Or feature size fits to inner block size of feature axis
+static inline bool check_redundant_1d_along_feature(layout const& l1, layout const& l2) {
+    if (!l1.data_padding && !l1.data_padding && l2.data_type == l1.data_type && l2.count() == l1.count()) {
+        // Get inner block size
+        if (static_cast<size_t>(l2.feature()) == l1.count() && l2.feature() == l1.feature() &&
+            static_cast<size_t>(l1.feature()) == l2.count() &&
+            (format::traits(l1.format).block_sizes.size() == 1 || format::traits(l2.format).block_sizes.size() == 1)) {
+            return true;
+        }
+
```

Comment:
What about bfyx [1,16,1,1] -> fsv16 [1,16,1,1]? Previous code returns true for such case, but new code will return false.

Reply:
Solution : added general function "format::is_multi_blocked" : use it